server/base/src/main/java/org/apache/accumulo/server/init/Initialize.java

```diff
@@ -748,24 +735,27 @@ private String getRootUserName(SiteConfiguration siteConfig, Opts opts) throws I
     if (opts.cliPassword != null) {
       return opts.cliPassword.getBytes(UTF_8);
     }
-    String rootpass;
-    String confirmpass;
+    char[] rootpass;
+    char[] confirmpass;
+    String strrootpass;
+    String strconfirmpass;
     do {
-      rootpass = getLineReader().readLine(
-          "Enter initial password for " + rootUser + getInitialPasswordWarning(siteConfig), '*');
+      rootpass = System.console().readPassword(
+          "Enter initial password for " + rootUser + getInitialPasswordWarning(siteConfig));
       if (rootpass == null) {
         System.exit(0);
       }
-      confirmpass =
-          getLineReader().readLine("Confirm initial password for " + rootUser + ": ", '*');
+      confirmpass = System.console().readPassword("Confirm initial password for " + rootUser + ":");
       if (confirmpass == null) {
         System.exit(0);
       }
-      if (!rootpass.equals(confirmpass)) {
+      strrootpass = new String(rootpass);
+      strconfirmpass = new String(confirmpass);
+      if (!strrootpass.equals(strconfirmpass)) {
         log.error("Passwords do not match");
       }
     } while (!rootpass.equals(confirmpass));
-    return rootpass.getBytes(UTF_8);
+    return strrootpass.getBytes(Charset.forName("UTF-8"));
```

Comment:
```suggestion
    return strrootpass.getBytes(UTF_8);
```

Reply:
If I do strrootpass.getBytes(UTF_8) It fails the quality check. Recommend way is the way I have it  strrootpass.getBytes(Charset.forName("UTF-8"));

Reply:
The tests pass for me with this change included, perhaps something else caused the test failures.

Reply:
I recommended this change since that was there previously and IntelliJ actually gives a warning with the way you have it. We are already importing the StandardCharset for UTF-8 and using it elsewhere so this part should follow the precedent already set.